src/main/java/org/openrefine/extensions/commons/importer/FileRecordToRows.java

```diff
@@ -41,7 +41,7 @@ public FileRecordToRows(Iterator<FileRecord> iteratorFileRecords, boolean catego
         this.identifierSpace = "https://commons.wikimedia.org/entity/";
         this.schemaSpace = "http://www.wikidata.org/prop/direct/";
         this.service = "https://commonsreconcile.toolforge.org/en/api";
-        this.reconConfig = new StandardReconConfig(service, identifierSpace, schemaSpace, null, null, true, Collections.emptyList());
+        this.reconConfig = new StandardReconConfig(service, identifierSpace, schemaSpace, null, null, true, relatedCategoriesIndex, Collections.emptyList(), relatedCategoriesIndex);
```

Comment:
This looks like a default suggestion of your IDE, which used whatever integer was in scope to provide a missing integer parameter for the constructor - that can work for more complex types but it's rarely the right thing to do for basic types like integers. Specifically, those two integer parameters are:
* batchSize -> I would leave it to 10
* limit -> I would leave it to 0

Reply:
You're right. I was a bit quick to accept the suggestion. I found another constructor that didn't specify limit, giving it the default, so I used that one.